src/TerminalSettings/TerminalSettings/SettingsControlViewModel.cpp

```diff
@@ -0,0 +1,22 @@
+#include "pch.h"
+#include "SettingsControlViewModel.h"
+#include "SettingsControlViewModel.g.cpp"
+
+namespace winrt::SettingsControl::implementation
+{
+    SettingsControlViewModel::SettingsControlViewModel()
+    {
+        m_homegriditem = winrt::make<SettingsControl::implementation::HomeGridItem>(L"");
```

Comment:
?

Reply:
I tried removing this line and it broke so I'm leaving it

Reply:
Jk I must have removed something else. I can take it out haha